config/config.go

```diff
@@ -34,9 +34,9 @@ func Get() (*Config, error) {
 	}
 
 	if cfg.Debug {
-		cfg.PatternLibraryAssetsPath = "http://localhost:9000/dist"
+		cfg.PatternLibraryAssetsPath = "http://localhost:9001/dist/assets"
```

Comment:
Incorrect local port number
```suggestion
		cfg.PatternLibraryAssetsPath = "http://localhost:9002/dist/assets"
```